recipe/meta.yaml

```diff
@@ -19,6 +19,8 @@ build:
   skip: true # [skip_cuda_prefect and (gpu_variant or "").startswith('cuda')]
   skip: true # [s390x]
   skip: true # [gpu_variant == "cuda-11"]
+  # do not mix cublas and mkl/openblas
+  skip: true # [((gpu_variant or "").startswith('cuda') and (blas_impl != "cublas")) or (not (gpu_variant or "").startswith('cuda') and (blas_impl == "cublas"))]
```

Comment:
this could be specified in the cbc? I think the blas_impl selectors could be made more mutually clear there anyway

Reply:
I did not find an easy way to express it. I will leave it like this for now.